docs/source/tutorials/code_snippets/event_loop_wx.py

```diff
@@ -8,4 +8,4 @@ class Counter(HasTraits):
 
 
 Counter().edit_traits()
-wx.PySimpleApp().MainLoop()
+wx.App().MainLoop()
```

Comment:
nitpicky comment - we usually try to leave an empty line at the end of all files. This is part of the enthought coding style.